### DISCUSSION TITLE
Cannot create a pull request

### DISCUSSION THREAD

**Discussion by emmayylu** [argoproj/argo-helm#1643] [Q&A] [answered]
Hello! I was trying to create a PR to make changes to the argo-events chart. When I ran `git push <branch>`. I got an error `fatal: unable to access 'https://github.com/argoproj/argo-helm.git/': The requested URL returned error: 403`.

**Comment by pdrastil** [comment, marked as answer]
Hi,
you cannot push to upstream repository as you are not maintainer. Proper process is to create your own fork and open PR against the upstream.
